src/mission_companion.cpp

```diff
@@ -185,28 +185,28 @@ bool talk_function::outpost_missions(npc *p, std::string id, std::string title)
     std::string entry, entry_aux;
 
     if (id == "SCAVENGER"){
-        col_missions["Assign Scavenging Patrol"] = "Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
+        col_missions["Assign Scavenging Patrol"] = _("Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
```

Comment:
BevapDin, thanks for a clue about gettext. I've installed gettext & python on windows and tried to run mentioned script in git-bash console but it gave me some errors, so I've run commands from the script manually and received output pot-file. Oh god, it works!